build-docker-image/action.yml

```
@@ -18,6 +18,11 @@ inputs:
     description: Relative path to the Dockerfile (Default = ./Dockerfile)
     default: ./Dockerfile
 
+outputs:
+  tag:
+    description: Docker image unique tag
+    value: ${{ steps.set-tag-output.outputs.tag }}
```

```suggestion
    value: ${{ env.IMAGE_TAG }}
```

Does this work instead of needing the additional step?

---

I tried but the output was empty. env may be resolved before the step is executed